plat/kvm/arm/lcpu.c

```diff
@@ -79,3 +79,13 @@ void ukplat_lcpu_irqs_handle_pending(void)
 {
 	// TODO
 }
+
+void ukplat_lcpu_set_auxsp(__uptr auxsp)
```

Comment:
This looks like an `ukarch_lcpu_*()` interface...

Reply:
We do not have such API's. Not sure I understand? This is the function that you previously asked me to rename:
`lcpu_arh_get/set_auxsp` -> `ukplat_lcpu_get/set_auxsp`.

Reply:
Okay, then leave it here. We will touch this code anyways again with re-arch in mind.